### DISCUSSION TITLE
I can't find my chia! I also can't farm

### DISCUSSION THREAD

**Discussion by Artemkos** [Chia-Network/chia-blockchain#17224] [Support]
Hello, please help! I can't find my chia! There should have been payouts, but my coins are not there! 
it just doesn't work after updating the program to version 2.0.x
please help!

![Скриншот 05-01-2024 151401](https://github.com/Chia-Network/chia-blockchain/assets/88540193/900d72c5-56b7-4db7-bae3-27314344a130)
![Скриншот 05-01-2024 151355](https://github.com/Chia-Network/chia-blockchain/assets/88540193/e1a73b29-f269-4e34-a5ad-261bb0cb7f0d)
![Скриншот 05-01-2024 151349](https://github.com/Chia-Network/chia-blockchain/assets/88540193/66ca5a9c-6794-4bb6-bc79-54641a6ab56d)
![Скриншот 05-01-2024 151319](https://github.com/Chia-Network/chia-blockchain/assets/88540193/984a250f-211b-46b3-a8e5-accd41becbfc)
![Скриншот 05-01-2024 151756](https://github.com/Chia-Network/chia-blockchain/assets/88540193/2d4b42cd-5eed-4f9b-8751-c2aa7bc99944)
![Скриншот 05-01-2024 151436](https://github.com/Chia-Network/chia-blockchain/assets/88540193/e81bfda8-1a11-4758-8630-fb761fbf361e) -

**Comment by Artemkos** [comment]
but in version 2.1.3, it's empty - there are no my pools!




![Скриншот 05-01-2024 164539](https://github.com/Chia-Network/chia-blockchain/assets/88540193/34b3c1b8-8c00-4596-97f9-82b0ad00554b)


![Скриншот 05-01-2024 164530](https://github.com/Chia-Network/chia-blockchain/assets/88540193/40732bd1-c148-4e8b-b8a3-70cec80ffc2f)

![Скриншот 05-01-2024 164506](https://github.com/Chia-Network/chia-blockchain/assets/88540193/c9b76438-c556-4bec-900c-7650510b8d02)

**Reply by BrandtH22** [reply]
Hey, as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

It seems we have a number of things occurring here:
- nodes not in sync, these last three images show clients that have nodes and wallets which are not in sync
- updates needed to the config file and a resync of the wallet needed when upgrading from a 1.x version to a 2.x version (instructions below)

I would first start with updating all machines as follows:
1. upgrade to the latest release candidate (there are peer related optimizations): https://github.com/Chia-Network/chia-blockchain/releases/tag/2.1.4-rc2

2. While installing that I would remove the current peers.dat file in ~\.chia\mainnet\db\peers.dat (this will force the client to connect to new peers upon restarting)

3. Once the latest RC is installed, have a new config file created:
 - Stop chia
 - Rename the config file at .chia\mainnet\config\config.yaml
 - Run chia init
 - Copy any custom settings from the old config to the new one (plot directories, pool information, database path, trusted nodes etc) - there are more automated ways to migrate that info if not comfortable editing the config file, yaml is very picky about formatting.
 - Start chia and let it sync
 
 If the issues persist after these updates feel free to let us know here or in our discord server.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
